### DISCUSSION TITLE
No files to upload. In case you have filled EXTENSION_FILTER, then check if all files have those extensions or not.

### DISCUSSION THREAD

**Discussion by DailyPlanetX** [anasty17/mirror-leech-telegram-bot#1712] [Q&A]
Hello, 
I try to download a file using a premium account with **/jdleech** (jdownloader command) but from bot I get this error: **No files to upload. In case you have filled EXTENSION_FILTER, then check if all files have those extensions or not.**

I don't understand well how to solve because I don't know where is this "EXTENSION_FILTER"

I used `/jdleech LINK -up b:@username`

Example link: `https://xxxxx.com/xxx71d96f `

![JDBOT](https://i.imgur.com/c1I2Rid.png)

I add correctly account from my.jdownloader.org

![JDBOT2](https://imgur.com/WK9oqvt.png)

I check config_sample.env for EXTENSION_FILTER but is empty so I don't understand the problem

![filter](https://imgur.com/N6rnMMF.png)

**EDIT**: maybe is due that I use a free account to download a premium file ?

![premium](https://imgur.com/5L9XI0G.png)

**Comment by anasty17** [comment]
reach me in group and send log file

**Reply by DailyPlanetX** [reply]
ok, I wrote to you in the group
